src/content/docs/onboarding/6-slack.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Slack channels
+description: Where we're talking
+---
+
+## General channels
+- `#engineering-all` Broadly for all engineers (internal and external) to share about engineering updates, questions, and more. Our most-used channel
+- `#engineering-hot-takes` Our fun channel for posting your engineering-related hot takes. No NJ-project-specific hot takes please
```

Comment:
[dust] If we're committed to making the docs public, maybe we don't want to advertise this publicly?